python_modules/dagster/dagster/_core/definitions/observe.py

```diff
@@ -12,22 +15,27 @@
     from ..execution.execute_in_process_result import ExecuteInProcessResult
 
 
+@deprecated_param(
```

Comment:
This isn't publicly exposed yet, right? If so, we don't need to worry about breaking changes.

Reply:
ah good point I'll remove the deprecation